scripts/zones/Carpenters_Landing/IDs.lua

```diff
@@ -28,9 +28,12 @@ zones[xi.zone.CARPENTERS_LANDING] =
         FISHING_MESSAGE_OFFSET        = 7285, -- You can't fish here.
         LOGGING_IS_POSSIBLE_HERE      = 7402, -- Logging is possible here if you have <item>.
         BEUGUNGEL_SHOP_DIALOG         = 7434, -- Hello, [sir/ma'am]! I'm selling goods direct from the Carpenters' Guild.
+        NOTHING_OUT_OF_THE_ORDINARY   = 7483, -- There is nothing out of the ordinary here.
+        STENCH_OF_DECAY               = 7484, -- You are overwhelmed by the putrid stench of decay!
         CRYPTONBERRY_EXECUTOR_DIE     = 7486, -- ...Cleave our foesss with barren hate.
         CRYPTONBERRY_ASSASSIN_2HR     = 7487, -- ..Take up thy lanternsss. The truth we shall illuminate.
         CRYPTONBERRY_EXECUTOR_2HR     = 7488, -- Through this we ssseek our just reward...
+        POLISH_MUSHROOM_SPORE         = 7489, -- You polish the <ki> with the glowing mushroom spores!
```

Comment:
Where have you got `<ki>` from, we don't use that anywhere apart from a comment in the middle of a single Windurst quest. These are the wildcards we use for matching (these comments are very important for the client update process): https://github.com/LandSandBoat/UpdateExtractor/blob/main/zone_texts.py#L81-L113

Reply:
I coded the quest in ASB first due to some issues getting retail installed. ASB uses <ki>. Updated to <keyitem>.